app.js

```diff
@@ -12,56 +12,67 @@ var explorer = require('./')
 var $hyperdrive = document.querySelector('#hyperdrive-ui')
 var $shareLink = document.getElementById('share-link')
 
-var url = window.location.toString()
-var key = url.split('#')[1]
-updateShareLink()
-var archive = drive.createArchive(key, {live: true})
-swarm(archive)
+var [, key, file] = window.location.hash.substr(1).match('([^\/]+)(/?.*)')
```

Comment:
you can pick the parts of babel that you need but in my experience it's not worth it yet, just slows you down too much